src/libs/ReactI13n.js

```diff
@@ -36,29 +38,24 @@ if (IS_CLIENT) {
 class ReactI13n {
   constructor(options = {}) {
     debug('init', options);
-    this._i13nNodeClass = typeof options.i13nNodeClass === 'function' ? options.i13nNodeClass : I13nNode;
-    this._plugins = {};
     this._eventsQueues = {};
-    this._isViewportEnabled = options.isViewportEnabled || false;
+    this._plugins = {};
     this._rootModelData = options.rootModelData || {};
+
     this._handlerTimeout = options.handlerTimeout || DEFAULT_HANDLER_TIMEOUT;
-    this._scrollableContainerId = options.scrollableContainerId || undefined;
+    this._i13nNodeClass = typeof options.i13nNodeClass === 'function' ? options.i13nNodeClass : I13nNode;
+    this._isViewportEnabled = options.isViewportEnabled || false;
+    this._scrollableContainerId = options.scrollableContainerId;
+    this._i13nInstance = null;
   }
 
-  /**
-   * Get ReactI13n Instance
-   * @method getInstance
-   * @return the ReactI13n instance
-   */
-  static getInstance() {
```

Comment:
this is for people accessing outside of React tree, moving to another export method and change not to pollute `window`